server/viame_tasks/tasks.py

```diff
@@ -1,16 +1,21 @@
 import json
 import os
-import re
 import tempfile
-from subprocess import Popen, PIPE
+from subprocess import Popen, PIPE, DEVNULL
+from pathlib import Path
 
 from girder_worker.app import app
-from girder_worker.utils import JobStatus
+
 
 class Config:
     def __init__(self):
-        self.pipeline_base_path = os.environ.get('VIAME_PIPELINES_PATH', '/opt/noaa/viame/configs/pipelines/')
-        self.viame_install_path = os.environ.get('VIAME_INSTALL_PATH', '/opt/noaa/viame')
+        self.pipeline_base_path = os.environ.get(
+            "VIAME_PIPELINES_PATH", "/opt/noaa/viame/configs/pipelines/"
+        )
+        self.viame_install_path = os.environ.get(
+            "VIAME_INSTALL_PATH", "/opt/noaa/viame"
+        )
```

Comment:
Looks like you and I are using different editor configurations.  We should probably set up pyproject.toml and black so that large diffs caused by linter settings don't happen.

In general, it's probably not good to have linting changes unrelated to your PR here.  

Do you know what config you're using that caused this?

Reply:
I'm just using the standard black configuration. I agree a `pyproject.toml` would be a good idea.